handbook/ops/roles/business_operations_analyst.md

```diff
@@ -0,0 +1,50 @@
+# Business Operations Analyst
+
+Sourcegraph’s business operations team empowers every part of the business with the data and insights, execution and project management, and strategic and business expertise that they need to make better decisions and move faster. As an early hire of the business operations team, you will be a vital member of both the business operations team and the company as we grow.
+
+Our team and our sales are growing fast—we’ve proven strong product-market fit, with wall-to-wall usage and top-tier engagement, genuine love from our users and customers, and tens of thousands of paid daily active users at some of the world’s premier development teams.
+
+## Responsibilities
+
+Your day-to-day responsibilities will include:
+* Partner with teams across the company to identify key metrics and growth drivers, build and maintain data pipelines and dashboards, and recommend actions to drive growth.
+* Perform analyses and research that contribute to important strategic and operational decisions
+* Design and implement the business systems and processes that will scale with as the company does.
```

Comment:
```suggestion
* Design and implement the business systems and processes that will scale as the company does.
```

Reply:
Not 100% sure, what is this bullet referring to? Any examples?

Reply:
Really could be anything. A new process for tracking goals, spinning up [DesignOps](https://firstround.com/review/the-case-for-adding-designops-to-your-org-chart-lessons-from-invision/), setting up A/B testing, etc. Kind of a general catch-all bullet that Ops is